### ISSUE TITLE
Tablet support

### ISSUE THREAD

**Issue by rexim** [tsoding/boomer#30]

**Comment by bresilla** [comment]
How cool would it be if something like this would be implemented!!

Using mouse to draw and make small notations. I follow this youtuber (image below) and when he explains a concept, uses this tool to annotate above (tablet or mouse).

Maybe this is what you have meant with "tablet support", i couldn't follow all your streams during the development so far. Or maybe this is just a far fetched idea.

![image](https://imgur.com/jAjQXPd.png)

**Comment by rexim** [comment]
@bresilla yes, I have something like that in mind. But we gonna start small. First we gonna just implement tablet navigation. Drawing stuff will be the next step :)

**Comment by mrgaturus** [comment]
2 years ago i added xinput2 to x11 nim wrappers, also there is a pen pressure example
